install.sh

```
@@ -0,0 +1,46 @@
+# https://www.mediawiki.org/wiki/Download_from_Git#Fetch_external_libraries
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)